account_invoice_supplier_ref_reuse/models/account_invoice.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    @api.multi
+    def _check_duplicate_supplier_reference(self):
+        # Do nothing instead of checking if the reference number already
```

Comment:
And when you uninstall the module, would raise errors for each of the invoice in with the same reference, isn't easier to update the reference with reference +"-a number" ex you have on test ABC123 first invoice, second one to be ABC123-1, third ABC123-2 and so on..

Reply:
Hi @feketemihai thanks for the feedback!
Having unique reference would indeed be easier but in some cases it's not really an option, as in the swiss example quoted in the description above. For such cases it's likely that people won't uninstall the module ever, since they need it to reuse some references anyway.